tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -270,6 +271,52 @@ describe(__filename, () => {
     }));
   });
 
+  it('lets an admin reply to a review', () => {
+    dispatchSignInActions({ store, permissions: [ALL_SUPER_POWERS] });
+
+    const addon = createInternalAddon(fakeAddon);
+    const review = _setReview({
+      ...fakeReview,
+      addon: {
+        id: addon.id,
+        slug: addon.slug,
+      },
+    });
+
+    const fakeDispatch = sinon.stub(store, 'dispatch');
+    const root = render({ addon, review });
+
+    const editButton = root.find('.AddonReviewListItem-begin-reply');
+    expect(editButton).toHaveLength(1);
+
+    const clickEvent = createFakeEvent();
+    editButton.simulate('click', clickEvent);
```

Comment:
Since the click handler doesn't have any logic to check for an admin, I don't think it's necessary to fire a click event fort this test. I know it makes the feature coverage a bit more thorough but I don't think it would really be that helpful.